### ISSUE TITLE
AC Console: make more tags available in the reviewer reminder modal 

### ISSUE THREAD

**Issue by zbialecki** [openreview/openreview-py#783]
Like the tag [[SUBMIT_REVIEW_LINK]], it would be helpful if we could make more tags available to the ACs. The following were proposed during ICLR2019:

1. Name of the recipient and the sender
2. Title of the paper

![image](https://user-images.githubusercontent.com/86174/97745706-a852b380-1abf-11eb-84fe-42b6c4e75a77.png)

**Comment by melisabok** [comment]
I would like to implement at least the `{name}` token in the backend. So we can send multiple messages in the same request call.

**Comment by zbialecki** [comment]
I don't think we added tags for paper title, but we add the backend tags for name and fullname. It's up to you if you think more frontend tags should be added or not.
